### PR TITLE
Data team helper to conflict resolver filter broken issue

### PR DESCRIPTION
**Pull request by sruthymathew123**
IN DTH, when you click an instrument under the data conflict tab it should take you to that specific conflict. Instead, it prepopulates only the DCCID but does not filter.

Discussed with Dave and the issue was that the conflict_resolver filter only works with post. But in DTH, it just passes the values via the URL. So it won't filter the results in conflict resolver. Altered the code to pass the value via POST so that filter works!!!

### CODE REVIEW

File: modules/data_team_helper/jsx/behavioural_qc_module.js [hunk 1]
```diff
@@ -143,6 +143,32 @@ var IncompleteCandidatesRow = React.createClass({
 });
 
 var InstrumentConflictsRow = React.createClass({
+    handleClick: function handleClick(event) {
```
Comment (driusan) [inline modules/data_team_helper/jsx/behavioural_qc_module.js]: This appears to be needless duplication. Can this use a closure returned from `loris.loadFilteredMenuClickHandler` instead?

File: modules/data_team_helper/jsx/behavioural_qc_module.js [hunk 2]
```diff
@@ -143,6 +143,32 @@ var IncompleteCandidatesRow = React.createClass({
...
+        event.preventDefault();
+        var row = this.props.row;
+        var candID= row.CandID;
+        var instrument=row.TableName;
+        var question = row.FieldName;
+        var form = $('<form />', {
```
Comment (davidblader) [inline modules/data_team_helper/jsx/behavioural_qc_module.js]: any reason in particular why you're not using `loris.loadFilteredMenuClickHandler` here?

**Comment by sruthymathew123** [comment]
@driusan Can you check now?

File: modules/data_team_helper/jsx/behavioural_qc_module.js [hunk 3]
```diff
@@ -164,14 +164,12 @@ var InstrumentConflictsRow = React.createClass({
...
-          <a
-            href={baseURL + "/conflict_resolver/?CandID=" + row.CandID}
-            className="conflict_resolver_link" data-pscid={row.PSCID}
-            data-question={row.FieldName} data-instrument={row.TableName}
-            data-visits={row.visit_label}
-          >
-            {row.test_name_display}
-          </a>
+          <a href="#" onClick={loris.loadFilteredMenuClickHandler(
+                'main.php?test_name=conflict_resolver',
```
Comment (driusan) [inline modules/data_team_helper/jsx/behavioural_qc_module.js]: this should still be baseURL + "/conflict_resolver/", not main.php..

**Comment by sruthymathew123** [comment]
@driusan can you verify the changes?

**Pull request merged**